src/scheduler/batch/main.ts

```diff
@@ -38,34 +38,42 @@ export async function populateCurrentAnnualReview(
       logger.info(`No list items identified for list ${list.id}, excluding from sending annual review emails`);
     } else {
       const listItemIdsForAnnualReview = listItemsEligibleForAnnualReview.map((listItem) => listItem.id);
+
+      const contexts = helpers.getDateContexts(list.nextAnnualReviewStartDate);
       const currentAnnualReview = getCurrentAnnualReviewData(listItemIdsForAnnualReview, contexts);
-      await updateListForAnnualReview(list, { currentAnnualReview });
+      let isUpdateList = true;
```

Comment:
minor, but point on readability/understandability so you can leave it for now, but  `shouldUpdateList` is probably better since it describes/determines what might happen next. `is` is better for current states. e.g. `isAnnualReview` `isValid`